Source/WebCore/css/parser/CSSPropertyParserHelpers.cpp

```diff
@@ -8281,6 +8281,8 @@ RefPtr<CSSValue> consumeCounterStyleAdditiveSymbols(CSSParserTokenRange& range,
             if (!integer)
                 return nullptr;
         }
+        if (!symbol)
+            return nullptr;
```

Comment:
Can we check for !symbol before checking for !integer and remove the !symbol check inside !integer?

Reply:
the symbol and weight can appear in any order [ [<integer [0,∞]>](https://www.w3.org/TR/css-values-4/#integer-value) [&&](https://www.w3.org/TR/css-values-4/#comb-all) [<symbol>](https://www.w3.org/TR/css-counter-styles-3/#typedef-symbol) ][#](https://www.w3.org/TR/css-values-4/#mult-comma)